zipline/finance/slippage.py

```diff
@@ -145,8 +146,8 @@ def process_order(self, trade_bar, order):
                 return
 
         return create_transaction(
-            trade_bar,
             order,
+            bar_data.current_dt,
```

Comment:
we could, but I think it adds a lot of power for the user to have access to `bar_data` when building a custom slippage function.  They can do `history` calls, check volatility, etc.